modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSetResultsSetTotalMethodCheck.java

```diff
@@ -5,79 +5,176 @@
 
 package com.liferay.source.formatter.check;
 
+import com.liferay.petra.string.StringPool;
 import com.liferay.portal.kernel.util.StringUtil;
 import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /**
  * @author Tamyris Bernardo
  */
-public class UpgradeSetResultsSetTotalMethodCheck extends BaseFileCheck {
+public class UpgradeSetResultsSetTotalMethodCheck extends BaseUpgradeCheck {
 
 	@Override
-	protected String doProcess(
+	protected String format(
 			String fileName, String absolutePath, String content)
 		throws Exception {
 
-		if (!fileName.endsWith(".java") && !fileName.endsWith(".jsp") &&
-			!fileName.endsWith(".jspf")) {
+		if (fileName.endsWith(".java")) {
+			content = _formatJava(content, fileName);
+		}
+		else if (fileName.endsWith(".jsp") || fileName.endsWith(".jspf")) {
+			content = _formatContent(content, content, fileName);
+		}
 
-			return content;
+		return content;
+	}
+
+	@Override
+	protected String[] getValidExtensions() {
+		return new String[] {"java", "jsp", "jspf"};
+	}
+
+	private String _formatContent(
+		String content, String fileContent, String fileName) {
+
+		String newContent = fileContent;
+
+		Matcher matcher = _setResultsPattern.matcher(content);
+
+		int setResultIndex = 0;
+
+		String setResultMethodCall = null;
```

Comment:
@TamyrisTorres, shouldn't we name these variables like `setResultsIndex` and `setResultsMethodCall`?

After all, the method associated to this check is actually `setResults` in plural instead of singular

Reply:
resolved.